components/budget/MissingBudget.tsx

```diff
@@ -14,6 +15,7 @@ export interface MissingBudgetProps {
 }
 
 export function MissingBudget({ otherBudgets, triggerRefetch, passwordHash, month, year }: MissingBudgetProps) {
+    const nav = useNavigation();;;;;;;;;;;;
```

Comment:
Too many semicolons;

Reply:
I know how much you like semi-colons so I sprinkled in a few extra for you